### DISCUSSION TITLE
How to get datetime range

### DISCUSSION THREAD

**Discussion by raanglada** [grafana/grafana-json-datasource#175] [Q&A] [answered]
Hi Marcus, 

Firstable congrats! The plugin is amazing!!

I have been doing some tests with the plugin and I don't see in the request the datetime range that we select in the Grafana UI. Is there a way to get this date time range in the request.

Thanks in advance

**Comment by raanglada** [comment]
I just saw this, 

https://github.com/marcusolsson/grafana-json-datasource/discussions/77 
https://marcus.se.net/grafana-json-datasource/macros/

Could please explain a little bit how to use those macros or share a link that explain better.

Thanks

**Comment by raanglada** [comment, marked as answer]
I was able to use the macros, as params in the query.

**Reply by marcusolsson** [reply]
Glad to hear! Any ideas on how I can improve the documentation?

**Reply by raanglada** [reply]
We can provided an example. In my case I just added two params. See the below image. 

![image](https://user-images.githubusercontent.com/34556757/134734582-fa138868-0a16-4c7e-82e6-e26bfc39e374.png)
